python/pypi-assets/urlformatter.py

```diff
@@ -40,7 +40,9 @@ def main():
 def format_url(metadata, base_url):
     """Format the distribution urls in metadata to point to the distributions
     hosted on base_url"""
-    for distributions in chain(metadata["releases"].values(), [metadata["urls"]]):
+    release_urls = metadata.get("releases", {}).values()
+    url_urls = [metadata.get("urls", [])]
```

Comment:
Assuming `metadata["urls"]` is already a list (as the default suggests):
```suggestion
    url_urls = metadata.get("urls", [])
```